source/common/upstream/load_stats_reporter.cc

```diff
@@ -86,6 +88,9 @@ void LoadStatsReporter::sendLoadStatsRequest() {
   ENVOY_LOG(trace, "Sending LoadStatsRequest: {}", request_.DebugString());
   stream_->sendMessage(request_, false);
   stats_.responses_.inc();
+  if (message_.get()) {
```

Comment:
Can message_ every be nullptr here? From a quick glance it seems like not but I'm probably missing something.

Reply:
Hmm I see it seems like we send an initial load stat request, and then wait to be told about the interval in order to send again? Is that right? Some more comments might be good on the flow in this file (even though it's documented in eds.proto also).

Reply:
Yeah, this is weird pre-existing code.
I think before we get the message we can't have any clusters_ set so I'm not sure why we send it.  @htuch is this behavior safe to change?

Reply:
I think with gRPC the server will get confused if there isn't some kind of initial message. I might be wrong but I don't think it's correct to just open a stream and expect the other side to message first. So given that we have to send first (I think) this is probably correct.